radicle-cli/src/commands/node/control.rs

```diff
@@ -0,0 +1,54 @@
+use radicle::node::{Address, Handle as _, NodeId};
+use radicle::Node;
+
+use crate::terminal as term;
+
+pub fn start() -> anyhow::Result<()> {
+    todo!()
+}
+
+pub fn stop(node: Node) -> anyhow::Result<()> {
+    let spinner = term::spinner("Stopping the node...");
+    if let Err(err) = node.shutdown() {
+        spinner.finish();
+        term::warning(&format!("Error occurred while shutting down node: {err}"));
+    } else {
+        spinner.finish();
+        term::success!("Successfully shut down the node")
```

Comment:
```suggestion
```
I think it's redundant because we'll see the checkmark next to `Stopping the node...` already :)